src/main/java/com/assertthat/selenium_shutterbug/utils/web/Browser.java

```diff
@@ -111,6 +115,33 @@ public BufferedImage takeScreenshotEntirePage() {
         return combinedImage;
     }
 
+    public BufferedImage takeScreenshotEntirePageUsingChromeCommand() {
+        DriverCommandExecutor driver = null;
+        try {
+            driver = new DriverCommandExecutor((ChromeDriver) this.driver);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        int verticalIterations = (int) Math.ceil(((double) this.getDocHeight()) / this.getViewportHeight());
+        for (int j = 0; j < verticalIterations; j++) {
+            this.scrollTo(0, j * this.getViewportHeight());
+            wait(scrollTimeout);
+        }
+        Object metrics = driver.evaluate(FileUtil.getJsScript(ALL_METRICS));
+        driver.sendCommand("Emulation.setDeviceMetricsOverride", metrics);
+        Object result = driver.sendCommand("Page.captureScreenshot", ImmutableMap.of("format", "png", "fromSurface", true));
+        driver.sendCommand("Emulation.clearDeviceMetricsOverride", ImmutableMap.of());
```

Comment:
It doesn't look like we need to clearDeviceMetricsOverride, as we don't really changing any initial device metrics just width and height which wont influence further test flow, moreover clearDeviceMetricsOverride for some reason spoins further highlighting/bluring etc. elements (coordinates are shifted).